.golangci.yml

```diff
@@ -1,161 +1,176 @@
+version: "2"
 linters:
   enable:
     - asciicheck
     - bodyclose
     - depguard
     - dogsled
     - dupl
-    - errcheck
     - goconst
-    - gofmt
-    - goimports
-    - gosimple
-    - govet
-    - ineffassign
```

Comment:
most of these I'm okay with removing but unsure on all of them

are any of these being added back elsewhere below additions to the configs?

Reply:
the v2 golangci linter has a `migrate` command that automatically updates the config, and it deleted them